src/views/world/view_gravity.cpp

```diff
@@ -0,0 +1,43 @@
+#include "views/view.hpp"
+
+namespace big
+{
+	void view::gravity()
+	{
+		components::command_checkbox<"modifygravity">();
+
+		if (g.world.gravity.modify_gravity)
+		{
+			ImGui::SliderFloat("GRAVITY_LEVEL"_T.data(), &g.world.gravity.current_gravity, 0.f, 1000.f, "%.1f");
+
+			ImGui::Separator();
+			components::sub_title("GRAVITY_LUNAR_PRESETS"_T);
+
+			// non-planets
+			components::button("GRAVITY_MOON"_T, [] { g.world.gravity.current_gravity = 1.6f; });
```

Comment:
Refactor this to be a drop-down. There is no reason to have every single item be a button. You also do not need components::button, as you are just writing to something within the cheat's own memory and do not need access to the natives. This can be done within the normal ImGui callback scope.

Reply:
This layout was just to ease testing the different values. Didn't think it looked bad ingame so didn't change it but I'll switch it up for you guys, no problem!